src/index.css

```diff
@@ -0,0 +1,80 @@
+/* 
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
```

Comment:
Not sure about this... @jeromesimeon?

Reply:
Where does that file come from? Who authored it?

Reply:
I believe this (and others in some of the files) is from the boilerplate generated from `yo`, @algomaster99 ?

Reply:
@irmerk correct

Reply:
I think remove it, @jeromesimeon?

Reply:
yes I think so!